estimagic/inference/bootstrap.py

```diff
@@ -0,0 +1,97 @@
+import pandas as pd
+
+from estimagic.inference.bootstrap_ci import _check_inputs
+from estimagic.inference.bootstrap_ci import _concatenate_functions
```

Comment:
In general, do not import private functions. Make the public if they are used in multiple places or move them to the module where there are actually used.

Reply:
Done in #202